api/prisma/migrations/20220615211706_init/migration.sql

```diff
@@ -0,0 +1,47 @@
+-- CreateEnum
+CREATE TYPE "Status" AS ENUM ('PENDING', 'APPROVED', 'REJECTED', 'PROCESSING', 'FINALIZED', 'FAULTY');
+
+-- CreateEnum
+CREATE TYPE "ChangeType" AS ENUM ('CATEGORY_SCORE', 'ACCESSORY_UNLOCK');
+
+-- CreateTable
+CREATE TABLE "OnChainChangeProposal" (
+    "id" TEXT NOT NULL,
+    "communityMemberEthAddress" TEXT NOT NULL,
```

Comment:
I think we should maybe have 
- a CommunityMember or User table with an UUID and ethereum address identifier (like what you have below, but less fields)
- an OtherNames (think of a better table name) table that would have like `user_id | identifier | platform` use this for storing GitHub username and discord username 
- Some Stats table with like `user_id | stat_increase | platform | timestamp` so this could store each PR merged or call attended if we wanted totals, we sum the table. Or possibly it could store a record that we mutate.

Reply:
sorry if this was already specified in the docs somewhere and I missed reviewing it there: getting consensus in Notion or whatever would have been cheaper time wise. Lmk if you're not pumped to switch all this or just disagree :)

Reply:
just don't love the idea of mutating the `CommunityMember` table if we want to track username on a new platform or track some new stat

Reply:
that makes sense! i like the idea of having a stats table, makes it really easy to determine things like community call streaks and such. will implement those suggestions, i like them overall